src/core/Clipboard.cpp

```diff
@@ -92,5 +154,93 @@ namespace lmms::Clipboard
 		return( QString::fromUtf8( mimeData->data( mimeType( MimeType::StringPair ) ) ).section( ':', 1, -1 ) );
 	}
 
+	std::pair<QString, QString> decodeMimeData(const QMimeData* mimeData)
+	{
+		const QList<QUrl> urls = mimeData->urls();
+
+		QString type{"missing_type"};
+		QString value{};
```

Comment:
```suggestion
		QString value;
```

@messmerd suggested this change. It was incorrect to assume value isn't initialized when defined. This isn't the case with basic c++ types such as `float` and `int`.